### PR TITLE
Progress card mode

### PR DESCRIPTION
**Pull request by fpieretti**
Add determinate mode to progress bar and loading card
Change route from initiafy-card to card in DS example

### CODE REVIEW

File: projects/design-system-initiafy/src/lib/loading-card/loading-card.component.ts [hunk 1]
```diff
@@ -9,6 +9,8 @@ export class LoadingCardComponent implements OnInit {
   @Input() cssClass = '';
   @Input() size: 'small' | 'std' | 'medium' | 'big' | 'full' = 'std';
   @Input() isLoading: boolean = false;
+  @Input() mode: 'indeterminate' | 'determinate' = 'indeterminate';
+  @Input() value: number = null;
```
Comment (JGough-Initiafy) [inline projects/design-system-initiafy/src/lib/loading-card/loading-card.component.ts]: is this a number between 1 and 100?

Comment (fpieretti) [reply projects/design-system-initiafy/src/lib/loading-card/loading-card.component.ts]: yes

File: projects/design-system-initiafy/src/lib/progress-bar/progress-bar.component.ts [hunk 2]
```diff
@@ -7,10 +7,19 @@ import { Component, OnInit, Input } from '@angular/core';
 })
 export class ProgressBarComponent implements OnInit {
   @Input() isLoading = '';
+  @Input() mode: 'indeterminate' | 'determinate' = 'indeterminate';
+  @Input() value: number = null;
```
Comment (JGough-Initiafy) [inline projects/design-system-initiafy/src/lib/progress-bar/progress-bar.component.ts]:
```suggestion
  /** Percentage Completed (0 - 100) */
  @Input() value: number = null;
```

Comment (JGough-Initiafy) [reply projects/design-system-initiafy/src/lib/progress-bar/progress-bar.component.ts]: this will add the desciprtion to the docs

**Pull request merged**